tox.ini

```diff
@@ -36,3 +36,4 @@ python =
   3.9: py39
   3.10: py310
   3.11: py311
+problem_matcher = False
```

Comment:
this was needed to avoid https://github.com/ymyzk/tox-gh-actions/issues/113

tox should have some auto-detection, but it fails for some reason for me and I couldn't be bothered to find out why